product_multiple_barcodes/views/product_template_views.xml

```diff
@@ -68,4 +68,30 @@
         </field>
     </record>
 
+    <record id="product_product_view_form_multiply_barcode" model="ir.ui.view">
+        <field name="name">product.product.view.form.multiply.barcode</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_normal_form_view"/>
+        <field name="arch" type="xml">
+                <header position="inside">
+                    <button string="Update Barcode" type="action"
+                        name="%(product_multiple_barcodes.action_multiply_barcode_wizard)d"
+                        attrs="{'invisible': [('attribute_line_ids', '&lt;=', 0)]}"/>
```

Comment:
I think this line gives problems
- Let's imagine we have created Product Template with only 1 variant. On product template we see button, and that is ok.
- But when we go to variatn we also want to see this button. But we do not see it now! And that is bug. For templates with 1 variant we should be able to update barcode both from template and from variant.

I think to fix this properly, instead of placing it in header button
<img width="1430" alt="Screen Shot 2020-02-25 at 15 15 56" src="https://user-images.githubusercontent.com/2680904/75250828-f2252480-57e1-11ea-931e-32c179aa41c4.png">